### DISCUSSION TITLE
How to blend two images, and also, how to blend them in JzAzBz colourspace

### DISCUSSION THREAD

**Discussion by P-Star7** [ImageMagick/ImageMagick#6295] [Help]
### ImageMagick version

7.1.0-55

### Operating system, version and so on

Windows 10 64-bit

### Description

Hello, I am trying to blend two images with a 50/50 blend (so, the exact midpoint between the pixels on the two images).

I have tried

magick white.png blue.png -colorspace jzazbz -composite -colorspace srgb lightblue.png*

But it only creates an exact copy of blue.png, instead of a blended version. I have also tried futzing around with -blend, but have been unable to get that to even work at all. I have tried to piece it together from the command-line help because the only tutorials currently existing are for IM6, but I haven't been able to. Could I please have a full example of usage of -blend?

My second question is, does the -colorspace operator even work with -composite or -blend, in regards to using that colourspace's math to blend instead of SRGB math?

*(These two images are just one pixel apiece, one being #FFFFFF and another being #0000FF. They're easy to recreate, but here they are anyways.)
![white](https://user-images.githubusercontent.com/26371167/236325579-1dab94fa-1701-44b5-87df-a28295ff8e26.png)
![blue](https://user-images.githubusercontent.com/26371167/236325575-3e3bd283-a945-4ee6-b72d-76184cd9db28.png)

**Comment by snibgo** [comment]
To blend two images, you can use "`-compose blend`", like this:
```
magick xc:White xc:Blue -colorspace JzAzBz -compose blend -define compose:args=50 -composite -colorspace sRGB txt:

# ImageMagick pixel enumeration: 1,1,0,65535,srgb
0,0: (45139,43855,66444)  #B053AB4FFFFF  srgb(68.8779%,66.9189%,101.386%)
```

My pages at http://im.snibgo.com are mostly for Windows BAT, and mostly for IM v7.
